app/Http/Controllers/Admin/MonsterCrudController.php

```diff
@@ -205,12 +241,71 @@ public function setupShowOperation()
             'name'    => 'easymde',
             'label'   => 'Markdown (EasyMDE)',
             'type'    => 'markdown',
+            'tab'     => 'WYSIWYG Editors',
+        ]);
+
+        $this->crud->addColumn([
+            'name'    => 'ckeditor',
+            'type'    => 'ckeditor',
+            'label'   => 'Ckeditor',
+            'tab'     => 'WYSIWYG Editors',
+        ]);
+
+        $this->crud->addColumn([
+            'name'   => 'summernote',
+            'type'   => 'summernote',
+            'label'  => 'Summernote',
+            'tab'    => 'WYSIWYG Editors',
+        ]);
+
+        $this->crud->addColumn([
+            'name'   => 'tinymce',
+            'type'   => 'tinymce',
+            'label'  => 'TinyMCE',
+            'tab'    => 'WYSIWYG Editors',
+        ]);
+
+        $this->crud->addColumn([
+            'name'    => 'wysiwyg',
+            'type'    => 'wysiwyg',
+            'label'   => 'Wysiwyg',
+            'tab'     => 'WYSIWYG Editors',
+        ]);
+
+        $this->crud->addColumn([
```

Comment:
Do we have this features field ?

Reply:
Yes we have a `features` field of the `repeatable` type